src/ruby/yast/builtins.rb

```diff
@@ -146,9 +146,18 @@ def self.maplist(object, &block)
       end
     end
 
-    # - Removes element from a list
-    # - Remove key/value pair from a map
-    # - Remove item from term
+    # Remove *element* from a **copy** of *object*.
+    #
+    # @param object [::Hash,::Array,Yast::Term,nil] a container
+    # @param element [Object,Integer,nil]
+    #   Any key for Hash;
+    #   Integer for Array and Term;
+    #   terms are indexed from 1 (**one**)
+    # @return [Object,nil]
+    #   Returns *object* unchanged if either *object* or *element* is `nil`;
```

Comment:
this is not true as it returns copy if element is nil. So not unchanged. For nil object it it not interesting as nil is immutable.